share/p2p/discovery/discovery.go

```diff
@@ -48,8 +48,11 @@ type Discovery struct {
 	host      host.Host
 	disc      discovery.Discovery
 	connector *backoffConnector
+
 	// onUpdatedPeers will be called on peer set changes
 	onUpdatedPeers OnUpdatedPeers
+	// indicates whether the
+	advertise bool
```

Comment:
This is not used anyhow, also comments needs a fix

Reply:
I guess the idea was to avoid advertising all discoveries at once, but left unfinished

Reply:
It is used, good catch on the comment. I added advertise to the private options (where the callback func for peer manager exists as well)

Reply:
I see, it was used in the manager accessing the private field :sigh: